src/Microsoft.OData.Core/PublicAPI/net48/PublicAPI.Unshipped.txt

```diff
@@ -1,4 +1,7 @@
+Microsoft.OData.IContainerProvider

```

Comment:
I see this getting added to the public API surface, but I don't see where it is coming from. Is it necessary?

Reply:
Yes. It is used.

Reply:
It is in the `Microsoft.OData.Core` library

Reply:
I think the name of the interface could be updated instead of doing away with it. If in future we want to add the ServiceProvider to a new class then we can still reuse the interface.

Reply:
IServiceCollectionProvider